data/human/outfits.txt

```diff
@@ -402,7 +402,7 @@ outfit "Pilot's License"
 	category "Special"
 	cost 50000
 	thumbnail "outfit/license"
-	description "Without a pilot's license, you cannot purchase a ship on a Republic planet. And if you buy an unlicensed ship on another planet and then bring it into Republic space, you can be fined until you cough up the money for a license. Needless to say, this policy is not terribly popular with freelance merchant captains."
+	description "Decades ago, starship pilots used to be able to fly without any sort of license. However, recent Republic legislation has led to the creation of pilot's licenses, which still permit all the freedoms captains had previously... if you can cough up the money for it. Needless to say, this system is not terribly popular with freelance merchant captains."
```

Comment:
Quoting from the historical timeline:
> 3000: The galaxy is beginning to slip into decadence, with megacorporations like the Syndicate once again gaining power. **The Republic has made several unpopular decisions, including expensive licenses that make it hard to make a start as an independent pilot**, and interstellar trade tariffs that weigh most heavily on low-value commodities such as food and clothing. This, combined with the Parliament structure that gives almost no voice to sparsely populated worlds, has led to an increase of anti-Republic sentiment.

I think I'd lean more into the recency of this change. I'd also still use the term "policy" instead of "system," as that lines up with the fact that it's the doing of the legislative body of the Republic.

```suggestion
	description "For much of the Republic's nearly three century long history, being able to pilot your own starship was limited only by your ability to purchase one. But at the turn of the millennium in an attempt to increase tax revenue, the Republic Parliament passed a law requiring that all ship captains purchase a pilot's license, and for no small fee at that. Needless to say, this policy is not terribly popular with freelance merchant captains."
```